hudi-common/src/test/java/org/apache/hudi/common/util/TestSerializableSchema.java

```diff
@@ -58,7 +58,7 @@ public void testLargeSchema() throws IOException {
   private void verifySchema(Schema schema) throws IOException {
     SerializableSchema serializableSchema = new SerializableSchema(schema);
     assertEquals(schema, serializableSchema.get());
-    assertTrue(schema != serializableSchema.get());
```

Comment:
The deserialized schema should never have the same reference with the original schema, we should make the test pass.

Reply:
> The deserialized schema should never have the same reference with the original schema, we should make the test pass.

done.